src/libs/PersonalDetailsUtils.ts

```diff
@@ -24,9 +24,14 @@ Onyx.connect({
     },
 });
 
-function getDisplayNameOrDefault(passedPersonalDetails?: Partial<PersonalDetails> | null, defaultValue = '', shouldFallbackToHidden = true): string {
-    const displayName = passedPersonalDetails?.displayName ? passedPersonalDetails.displayName.replace(CONST.REGEX.MERGED_ACCOUNT_PREFIX, '') : '';
+function getDisplayNameOrDefault(passedPersonalDetails?: Partial<PersonalDetails> | null, defaultValue = '', shouldFallbackToHidden = true, shouldUsePostfix = false, postfix = Localize.translateLocal('common.you').toLowerCase()): string {
+    let displayName = passedPersonalDetails?.displayName ? passedPersonalDetails.displayName.replace(CONST.REGEX.MERGED_ACCOUNT_PREFIX, '') : '';
+    if (shouldUsePostfix && !!displayName) {
+        displayName = `${displayName} (${postfix})`;
+    }
```

Comment:
I was thinking about passing postfix and checking if it's non-empty instead of using 2 params
I see now that we'll have to pass this postfix in multiple places, so let's just revert back to your original solution to keep things YAGNI. Sorry about that.